source/extensions/omni.isaac.lab/omni/isaac/lab/envs/common.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+import torch
+from typing import Dict, Literal
+
+import omni.isaac.lab.envs.mdp as mdp
+from omni.isaac.lab.managers import EventTermCfg as EventTerm
+from omni.isaac.lab.utils import configclass
+
+##
+# Configuration.
+##
+
+
+@configclass
+class ViewerCfg:
+    """Configuration of the scene viewport camera."""
+
+    eye: tuple[float, float, float] = (7.5, 7.5, 7.5)
+    """Initial camera position (in m). Default is (7.5, 7.5, 7.5)."""
+
+    lookat: tuple[float, float, float] = (0.0, 0.0, 0.0)
+    """Initial camera target position (in m). Default is (0.0, 0.0, 0.0)."""
+
+    cam_prim_path: str = "/OmniverseKit_Persp"
+    """The camera prim path to record images from. Default is "/OmniverseKit_Persp",
+    which is the default camera in the viewport.
+    """
+
+    resolution: tuple[int, int] = (1280, 720)
+    """The resolution (width, height) of the camera specified using :attr:`cam_prim_path`.
+    Default is (1280, 720).
+    """
+
+    origin_type: Literal["world", "env", "asset_root"] = "world"
+    """The frame in which the camera position (eye) and target (lookat) are defined in. Default is "world".
+
+    Available options are:
+
+    * ``"world"``: The origin of the world.
+    * ``"env"``: The origin of the environment defined by :attr:`env_index`.
+    * ``"asset_root"``: The center of the asset defined by :attr:`asset_name` in environment :attr:`env_index`.
+    """
+
+    env_index: int = 0
+    """The environment index for frame origin. Default is 0.
+
+    This quantity is only effective if :attr:`origin` is set to "env" or "asset_root".
+    """
+
+    asset_name: str | None = None
+    """The asset name in the interactive scene for the frame origin. Default is None.
+
+    This quantity is only effective if :attr:`origin` is set to "asset_root".
+    """
+
+
+@configclass
+class DefaultEventManagerCfg:
+    """Configuration of the default event manager.
+
+    This manager is used to reset the scene to a default state. The default state is specified
+    by the scene configuration.
+    """
+
+    reset_scene_to_default = EventTerm(func=mdp.reset_scene_to_default, mode="reset")
```

Comment:
would rather copy this one in both, a bit more code but clearer

Reply:
Just realized. This is None by default for direct workflow. Fixed now.